libraries/c_sdk/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -19,7 +19,7 @@
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  *
- * http://aws.amazon.com/freertos
+ * https://github.com/freertos
  * http://www.FreeRTOS.org
```

Comment:
The FreeRTOS link also needs to be https.

Reply:
Addressed